### PR TITLE
Profile dashboard CDO information

### PR DESCRIPTION
**Pull request by mjoyce91**
Adds a static CDO section to the Profile Dashboard, for eventual integration with real data from the API. Also abstracts the green status circle and the profile picture to a common component, since those visual elements are starting to appear in multiple places.

Dependent on #898 and #899.

### CODE REVIEW

File: src/Components/BorderedList/BorderedList.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```
Comment (burgwyn) [inline src/Components/BorderedList/BorderedList.jsx]: I'm not seeing a test for this one

Comment (mjoyce91) [reply src/Components/BorderedList/BorderedList.jsx]: Added

File: src/actions/notifications.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+export function notificationsFetchData() {
+  return (dispatch) => {
+    axios.get(`${api}/notification/?limit=3`, { headers: { Authorization: fetchUserToken() } })
+            .then(({ data }) => data.results)
+            .then((notifications) => {
+              dispatch(notificationsFetchDataSuccess(notifications));
+              dispatch(notificationsIsLoading(false));
+              dispatch(notificationsHasErrored(false));
+            })
+            .catch(() => dispatch(notificationsHasErrored(true)));
```
Comment (burgwyn) [inline src/actions/notifications.js]: Do we want `dispatch(notificationsIsLoading(false));`?

File: src/actions/assignment.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+export function assignmentFetchData() {
+  return (dispatch) => {
+    axios.get(`${api}/profile/assignments/`, { headers: { Authorization: fetchUserToken() } })
+            .then(({ data }) => data.results[0] || {})
+            .then((assignment) => {
+              dispatch(assignmentFetchDataSuccess(assignment));
+              dispatch(assignmentIsLoading(false));
+              dispatch(assignmentHasErrored(false));
+            })
+            .catch(() => dispatch(assignmentHasErrored(true)));
```
Comment (burgwyn) [inline src/actions/assignment.js]: `dispatch(assignmentIsLoading(false));`?

Comment (mjoyce91) [reply src/actions/assignment.js]: Added

File: src/actions/notifications.js (hunk 2 above)
Comment (mjoyce91) [reply src/actions/notifications.js]: Added

**Pull request merged**